addons/httprox/common/src/main/java/org/commonjava/indy/httprox/handler/ProxyResponseWriter.java

```diff
@@ -238,13 +239,13 @@ private void handleError( final Throwable error, final HttpWrapper http )
         }
     }
 
-    private void transfer( final HttpConduitWrapper http, final RemoteRepository repo, final String path,
+    private Transfer transfer( final HttpConduitWrapper http, final RemoteRepository repo, final String path,
```

Comment:
Aha, this was from my previous changes. I used it to delete files in calling method. But it is not needed any more, since we use nocache decorator now. I reverted it.